src/acvm_interop/proof_system.rs

```diff
@@ -48,7 +48,7 @@ impl ProofSystemCompiler for Gnark {
     }
 
     fn get_exact_circuit_size(&self, circuit: &Circuit) -> u32 {
-        todo!()
+        gnark_backend::get_exact_circuit_size(circuit).unwrap()
```

Comment:
Just a comment here because it contradicts what I told you before, we are using `unwrap` in the implementation of this trait because it is foreign and its methods don't support returning results.

Reply:
Thank you! I did try to use `?` but I got the error you mention